numpy/core/src/umath/loops_umath_fp.dispatch.c.src

```diff
@@ -74,7 +92,11 @@ simd_@func@_f64(const double *src, npy_intp ssrc,
         } else {
             x = npyv_loadn_tillz_f64(src, ssrc, len);
         }
+        #if defined(_NPY_OPTIMIZED_ROUTINES)
+        npyv_f64 out = __v_@func@(x);
```

Comment:
In the pl/ subdir of AOR, and likely in math/ as well, the `__v_`-prefixed symbols are being removed (see for example [this commit](https://github.com/ARM-software/optimized-routines/commit/0f7044c217d976c89e5ff32af1bbb68e28f66358)). From now on the only reliably supported names are the [VFABI](https://github.com/ARM-software/abi-aa/blob/2982a9f3b512a5bfdc9e3fea5d3b298f9165c36b/vfabia64/vfabia64.rst#name-mangling-function)-mangled variants, so I don't think this line is future-proof

Reply:
Argh, this makes it slightly more irritating than the existing SVML integration, I can change these when I integrate the next set of functions though.